gradoop-flink/src/main/java/org/gradoop/flink/io/impl/dot/functions/DotFileFormatHtml.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.io.impl.dot.functions;
+
+import org.apache.commons.lang3.StringUtils;
+import org.gradoop.common.model.api.entities.EPGMElement;
+import org.gradoop.common.model.impl.pojo.Vertex;
+import org.gradoop.common.model.impl.properties.Properties;
+import org.gradoop.common.model.impl.properties.Property;
+import org.gradoop.flink.model.impl.layouts.transactional.tuples.GraphTransaction;
+
+import static org.apache.commons.lang3.StringEscapeUtils.escapeHtml4;
+
+/**
+ * Converts a GraphTransaction to the following .dot format:
+ * <p>
+ *   digraph 0
+ *   {
+ *   gradoopId1 [label=<<table>...</table>>];
+ *   gradoopId2 [label=<<table>...</table>>];
+ *   gradoopID3;
+ *   gradoopID4;
+ *   gradoopId1->gradoopId2 [label=<<table>...</table>>];
+ *   gradoopId2->gradoopId1 [label=<<table>...</table>>];
+ *   gradoopId3->gradoopId4;
+ *   }
+ * </p>
+ */
+public class DotFileFormatHtml extends AbstractDotFileFormat {
+
+  /**
+   * color color for header background and properties text
+   */
+  private String color;
+
+  /**
+   * Constructor
+   *
+   * @param printGraphHead true, iff graph head data shall be attached to the output
+   * @param color the color for header background and properties text
+   */
+  public DotFileFormatHtml(Boolean printGraphHead, String color) {
```

Comment:
Change the paramter type to `boolean`, as the `setPrintGraphHead` expects a `boolean`.